base58/src/lib.rs

```diff
@@ -1,19 +1,40 @@
 // SPDX-License-Identifier: CC0-1.0
 
-//! Base58 encoder and decoder.
+//! Bitcoin base58 encoding and decoding.
 //!
-//! This module provides functions for encoding and decoding base58 slices and
-//! strings respectively.
-//!
-
-use core::{fmt, iter, slice, str};
+//! This crate can be used in a no-std environment but requires an allocator.
```

Comment:
FTR, I'm pretty sure this can be improved later. I'd love to have non-allocating address.